### ISSUE TITLE
Transaction Date, Value, Memo

### ISSUE THREAD

**Issue by lowercase00** [jseutter/ofxparse#136]
Thanks for the tool, very useful!
I was wondering if there's the complete set of functions anywhere, I'm trying to export a report with the transaction date, the memo and the value... thanks a lot.

**Comment by jseutter** [comment]
No, there isn't a complete set of functions, and I wish we had one.  :(

You have a couple of options to find out how:
1. Use a debugger to step through the code (google can show you how to do this), then print(dir(<object name here>)) will print every attribute available on the transaction object.
2. Look in the source code.  ofxparse/ofxparse.py is where the code is located, and each class lists the attributes it has.  Eg:
```
class Transaction(object):
    def __init__(self):
        self.payee = ''
        self.type = ''
        self.date = None
        self.amount = None
        self.id = ''
        self.memo = ''
        self.sic = None
        self.mcc = ''
        self.checknum = ''
```

**Comment by cilynx** [comment]
I just submitted #138 to flesh out the attribute documentation a bit.
